.github/CODEOWNERS

```diff
@@ -728,6 +728,10 @@
 # ServiceLabel:  %Service Fabric
 # ServiceOwners:                                     @QingChenmsft @vaishnavk @juhacket
 
+# PRLabel: %Web PubSub
```

Comment:
Not sure whether the name be `WebPubSub`.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=4010122&view=logs&j=a460d732-23aa-5493-a411-cc406067acf8&t=2b56d21f-1cfb-5a3c-ae52-6b8b1a9ee09a

Do we also add a ServiceLabel/ServiceOwners for it?